### DISCUSSION TITLE
Eslint error import/no-anonymous-default-export

### DISCUSSION THREAD

**Discussion by nikoskleidis** [storybookjs/storybook#15034] [Help]
As mentioned on this issue 
https://github.com/storybookjs/storybook/issues/12914

this seems to be fixed during build but it fails during development. 
We have a pre-commit hook that runs `npx lint-staged` and it fails to pass successfully. 
Does anybody have a similar issue?

**Comment by yannbf** [comment]
This didn't work for you? https://github.com/storybookjs/storybook/issues/12914#issuecomment-717681952

**Reply by nikoskleidis** [reply]
Thanks @yannbf. This works for me yes. I thought that after 6.2.0 it would work without needing this.
